sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/TestJsonData.scala

```diff
@@ -214,6 +214,11 @@ private[json] trait TestJsonData {
       """{"a": {"b": 1}}""" ::
       """{"a": []}""" :: Nil)
 
+  def doubleRecords: RDD[String] =
+    sqlContext.sparkContext.parallelize(
+      s"""{"a": 1${"0" * 38}, "b": 0.01, "c": 92233720368547758070, "d": 1.01}""" ::
```

Comment:
Can we have a column that have different schema (range) on different row, they got merged together finally.